COLLADAMaya/README.md

```diff
@@ -1,68 +1,81 @@
-If you want to remove constant key frames in Bezier mode, use "Optimize Animations" options
-When you export Bezier animation, you will have animation value for translation X,Y,Z and Rotation X,Y,Z even if the value is constant
-When you use "Optimize Animations" in Bezier Animation mode, only useful (not constant value) from translation X,Y,Z or Rotation X,Y,Z will be exported
-
-	. Pose Animation:
-	
-If you want to export Pose Animation (only 1 key frame animation), be careful to set Source Start = Source End = 0 in Animation Clip Attributes in Maya
-
-	. Split animation/Mesh into two differents DAE files:
-	
-use Polygon Meshes and Skin and Joints options ( no Animations option) if you want to export only Mesh into one DAE
-use Animations and Joints options (no Polygon Meshes and no Skin option) if you want to export only Animation into one DAE
-if you want to export full animation and mesh into the same DAE, use Polygon Meshes and Skin and Joints and Animations options
-
-
-- Extra Attributes
-
-This plug-in currently supports export of Maya extra attributes for the following Maya node types:
-	. Transform (in <node>)
-	. Mesh shape (in <geometry>)
-	. Light shape (in <light>)
-	. PhysX RigidBody (in <rigid_body>)
-	. PhysX PhysicsShape (in <shape>)
-	. PhysX RigidConstraint (in <rigid_constraint>)
-
-Extra attributes are exported in <extra>/<technique profile="OpenCOLLADAMaya"> section of corresponding COLLADA element with the following syntax:
-
-<param name="attribute_name" type="attribute_type">attribute_value</param>
-
-	. attribute_name is the name of the extra attribute.
-	. attribute_type is the type of the extra attribute. Supported types are:
-		. "int" (Maya extra attribute type: Integer)
-		. "string" (Maya extra attribute type: String or Enum)
-		. "float" (Maya extra attribute type: Float)
-		. "bool" (Maya extra attribute type: Boolean)
-		. "double3" (Maya extra attribute type: Vector)
-	. attribute_value is the extra attribute value as text.
\ No newline at end of file
+COLLADAMaya is a new implementation of a Autodesk Maya plug-in to export a Maya scene or parts of it to a COLLADA file, released under the MIT-license.
+
+This plug-in does not store the COLLADA document in an intermidiate data model, it writes the data directly into file. This results in reduction of memory consumption and better performance.
+
+For installation instructions please read the "INSTALL" file.
+
+# General export options in OpenColladaMaya export plugin:
+
+## <source_data> path format
+	
+<source_data> element in exported DAE contains a URI reference to the source Maya file. This option allows to make URI absolute or relative to specific location:
+
+* ```Absolute```
+  * Path to source file is absolute.
+* ```Relative to exported DAE```
+  * Path to source file is relative to exported DAE file.
+* ```Relative to Maya project workspace```
```

Comment:
According to [RFC 3986 5.1](https://tools.ietf.org/html/rfc3986#section-5.1), this idea can only work if the .dae document establishes a base URI embedded in the document, as in  section 5.1.1. As every .dae document is retrieved via a URL, that means section 5.1.3 applies at the least. To implement this according to specifications, you need to use https://www.w3.org/TR/xmlbase/